tiatoolbox/cli.py

```diff
@@ -66,9 +66,18 @@ def main():
     "the meta information, default=show",
 )
 @click.option(
-    "--verbose", type=bool, default=True, help="Print output, default=True",
+    "--workers",
```

Comment:
We removed multiproc from develop branch, best to merge latest develop branch into this branch to avoid these issues.

Reply:
Fixed in [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/47deccd6060278762872c1930d14707625aa2b2d)